CefSharp.Example/DownloadHandler.cs

```diff
@@ -2,12 +2,23 @@
 //
 // Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
 
+using System;
+
 namespace CefSharp.Example
 {
     public class DownloadHandler : IDownloadHandler
     {
+        public event EventHandler<DownloadItem> OnBeforeDownloadFired;
+
+        public event EventHandler<DownloadItem> OnDownloadUpdatedFired;
+
         public void OnBeforeDownload(IBrowser browser, DownloadItem downloadItem, IBeforeDownloadCallback callback)
         {
+            if (this.OnBeforeDownloadFired != null)
```

Comment:
I update to match that style. Presumably using a null-conditional would also be okay? E.g.

`OnBeforeDownloadFired?.Invoke(this, downloadItem);`